testsuite/cluster_test/src/tx_emitter.rs

```diff
@@ -0,0 +1,330 @@
+use crate::{cluster::Cluster, instance::Instance};
+use admission_control_proto::proto::{
+    admission_control::SubmitTransactionRequest, admission_control_grpc::AdmissionControlClient,
+};
+use client::{AccountData, AccountStatus};
+use grpcio::{ChannelBuilder, EnvBuilder};
+use proto_conv::IntoProto;
+use std::{
+    env,
+    str::FromStr,
+    sync::Arc,
+    thread,
+    time::{Duration, Instant},
+};
+
+pub struct TxEmitter {
```

Comment:
nit: could you move it down? (it's hidden between the `use` directives right now.

Reply:
Oh lol, yeah, probably some mix of cargo fmt and me being sloppy